src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,270 @@
+                self._run()
+            except TestFailedError:
+                printTester(f"'{self.name}' failed.")
+            except Exception as e:
+                self.__markAsFailed(f"'{self.name}' had an internal error. {e}.\nPlease report this to an instructor!")
+                print_exc()
+                self._onFailed()
+
+        self.case.time = datetime.now() - startTime
+        self.suite.addCase(self.case)
+
+    def __checkTestRequirements(self, testResults: Dict[str, Result]):
```

Ruff

Missing return type annotation for private function `__checkTestRequirements`

(ANN202)